python/native_functions.py

```diff
@@ -212,6 +212,10 @@ def add_funcs(global_scope):
     global_scope.add_native_function(
         "print", [("val", print_generic)], print_generic, special_print
     )
+    print_with_end_generic = NGenericType("t")
+    global_scope.add_native_function(
+        "printWithEnd", [("val", print_generic), ("end", "str")], print_generic, special_print
```

Comment:
I suppose it would be ideal if `printWithEnd` returned a `cmd` since it has a side effect.

Also, `val` should be the last argument so that it can be used for currying; for example,

```ts
let printWithSpace = printWithEnd(" ")
printWithSpace("hi")
printWithSpace("hello")
printWithSpace("wow")
```

is more useful than

```ts
let printSheepThen = printWithEnd("🐑")
printSheepThen("\n")
```

Reply:
Ok, did that